### PR TITLE
docs: Fix Python taint tracking links

### PR DESCRIPTION
**Pull request by RasmusWL**
at some point we moved security/TaintTracking.qll to dataflow/TaintTracking.qll

### CODE REVIEW

**Comment by jf205** [comment]
Actually, ignore my approval, I think I just spotted a broken link.

File: docs/language/learn-ql/python/introduce-libraries-python.rst [hunk 1]
```diff
@@ -323,8 +323,8 @@ The CodeQL library for Python also supplies classes to specify taint-tracking an
 Summary
 ~~~~~~~
 
-- `TaintKind <https://help.semmle.com/qldoc/python/semmle/python/security/TaintTracking.qll/type.TaintTracking$TaintKind.html>`__
-- `Configuration <https://help.semmle.com/qldoc/python/semmle/python/security/TaintTracking.qll/type.TaintTracking$TaintTracking$Configuration.html>`__
+- `TaintKind <https://help.semmle.com/qldoc/python/semmle/python/dataflow/TaintTracking.qll/type.TaintTracking$TaintKind.html>`__
+- `Configuration <https://help.semmle.com/qldoc/python/semmle/python/dataflow/TaintTracking.qll/type.TaintTracking$TaintTracking$Configuration.html>`__
```
Comment (jf205) [inline docs/language/learn-ql/python/introduce-libraries-python.rst]:
```suggestion
- `Configuration <https://help.semmle.com/qldoc/python/semmle/python/dataflow/Configuration.qll/type.Configuration$TaintTracking$Configuration.html>`__
```

File: docs/language/learn-ql/python/taint-tracking.rst [hunk 2]
```diff
@@ -13,36 +13,36 @@ Taint tracking differs from basic data flow in that it considers non-value-prese
...
+2. One or more ``sinks``, to where the data or taint may flow, represented by the `TaintTracking::Sink <https://help.semmle.com/qldoc/python/semmle/python/dataflow/TaintTracking.qll/type.TaintTracking$TaintSink.html>`__ class.
+3. Zero or more ``sanitizers``, represented by the `Sanitizer <https://help.semmle.com/qldoc/python/semmle/python/dataflow/TaintTracking.qll/type.TaintTracking$Sanitizer.html>`__ class.
 
 A taint tracking or data flow query gives results when there is the flow of data from a source to a sink, which is not blocked by a sanitizer.
 
-These three components are bound together using a `TaintTracking::Configuration <https://help.semmle.com/qldoc/python/semmle/python/security/TaintTracking.qll/type.TaintTracking$TaintTracking$Configuration.html>`__.
+These three components are bound together using a `TaintTracking::Configuration <https://help.semmle.com/qldoc/python/semmle/python/dataflow/TaintTracking.qll/type.TaintTracking$TaintTracking$Configuration.html>`__.
```
Comment (jf205) [inline docs/language/learn-ql/python/taint-tracking.rst]: [](url)
```suggestion
These three components are bound together using a `TaintTracking::Configuration <https://help.semmle.com/qldoc/python/semmle/python/dataflow/Configuration.qll/type.Configuration$TaintTracking$Configuration.html>`__.
```

**Comment by RasmusWL** [comment]
Good that you spotted it! I fixed it and amended the commit, but it should all be good now :tada:

**Comment by jf205** [comment]
Thanks!

**Pull request merged**